### ISSUE TITLE
Error: No such command 'videogen'. 

### ISSUE THREAD

**Issue by evoluder** [brycedrennan/imaginAIry#456]
can't seem to reopen https://github.com/brycedrennan/imaginAIry/issues/400, ptal at my last comment

**Comment by evoluder** [comment]
decided to try this from scratch with new environment, same behavior as described on the other bug:

conda create --name imagine python=3.10
conda activate imagine
pip install torch torchvision torchaudio --index-url https://download.pytorch.org/whl/cu118
but then...

pip install imaginairy
Collecting imaginairy
  Using cached imaginAIry-14.0.4-py3-none-win_amd64.whl.metadata (36 kB)
Collecting click>=8.0.0 (from imaginairy)
  Downloading click-8.1.7-py3-none-any.whl.metadata (3.0 kB)
Collecting click-help-colors>=0.9.1 (from imaginairy)
  Using cached click_help_colors-0.9.4-py3-none-any.whl.metadata (4.1 kB)
Collecting click-shell>=2.0 (from imaginairy)
  Using cached click_shell-2.1-py2.py3-none-any.whl (8.6 kB)
Collecting protobuf!=3.19.5,!=3.20.2 (from imaginairy)
  Downloading protobuf-4.25.2-cp310-abi3-win_amd64.whl.metadata (541 bytes)
Collecting facexlib>=0.2.1.1 (from imaginairy)
  Using cached facexlib-0.3.0-py3-none-any.whl (59 kB)
Collecting fastapi>=0.70.0 (from imaginairy)
  Downloading fastapi-0.109.0-py3-none-any.whl.metadata (24 kB)
Collecting ftfy>=6.0.1 (from imaginairy)
  Using cached ftfy-6.1.3-py3-none-any.whl.metadata (6.2 kB)
Requirement already satisfied: torch>=2.1.0 in c:\users\anon\anaconda3\envs\imagine\lib\site-packages (from imaginairy) (2.1.2+cu118)
Requirement already satisfied: numpy<1.27.0,>=1.22.0 in c:\users\anon\anaconda3\envs\imagine\lib\site-packages (from imaginairy) (1.24.1)
Collecting tqdm>=4.64.0 (from imaginairy)
  Using cached tqdm-4.66.1-py3-none-any.whl.metadata (57 kB)
Collecting diffusers>=0.3.0 (from imaginairy)
  Using cached diffusers-0.25.1-py3-none-any.whl.metadata (19 kB)
Collecting imageio>=2.9.0 (from imaginairy)
  Downloading imageio-2.33.1-py3-none-any.whl.metadata (4.9 kB)
Requirement already satisfied: Pillow>=9.1.0 in c:\users\anon\anaconda3\envs\imagine\lib\site-packages (from imaginairy) (9.3.0)
Collecting psutil>5.7.3 (from imaginairy)
  Downloading psutil-5.9.7-cp37-abi3-win_amd64.whl.metadata (22 kB)
Collecting omegaconf>=2.1.1 (from imaginairy)
  Using cached omegaconf-2.3.0-py3-none-any.whl (79 kB)
Collecting open-clip-torch>=2.0.0 (from imaginairy)
  Using cached open_clip_torch-2.24.0-py3-none-any.whl.metadata (30 kB)
Collecting opencv-python>=4.4.0.46 (from imaginairy)
  Downloading opencv_python-4.9.0.80-cp37-abi3-win_amd64.whl.metadata (20 kB)
Collecting pydantic>=2.3.0 (from imaginairy)
  Downloading pydantic-2.5.3-py3-none-any.whl.metadata (65 kB)
     ---------------------------------------- 65.6/65.6 kB 3.5 MB/s eta 0:00:00
Requirement already satisfied: requests>=2.28.1 in c:\users\anon\anaconda3\envs\imagine\lib\site-packages (from imaginairy) (2.28.1)
Collecting jaxtyping>=0.2.23 (from imaginairy)
  Using cached jaxtyping-0.2.25-py3-none-any.whl.metadata (6.4 kB)
Collecting einops>=0.3.0 (from imaginairy)
  Using cached einops-0.7.0-py3-none-any.whl.metadata (13 kB)
Collecting safetensors>=0.4.0 (from imaginairy)
  Using cached safetensors-0.4.1-cp310-none-win_amd64.whl.metadata (3.8 kB)
Collecting scipy<1.11 (from imaginairy)
  Using cached scipy-1.10.1-cp310-cp310-win_amd64.whl (42.5 MB)
Collecting termcolor (from imaginairy)
  Using cached termcolor-2.4.0-py3-none-any.whl.metadata (6.1 kB)
Collecting timm!=0.9.0,!=0.9.1,>=0.4.12 (from imaginairy)
  Using cached timm-0.9.12-py3-none-any.whl.metadata (60 kB)
Collecting torchdiffeq>=0.2.0 (from imaginairy)
  Using cached torchdiffeq-0.2.3-py3-none-any.whl (31 kB)
Requirement already satisfied: torchvision>=0.13.1 in c:\users\anon\anaconda3\envs\imagine\lib\site-packages (from imaginairy) (0.16.2+cu118)
Collecting transformers>=4.19.2 (from imaginairy)
  Using cached transformers-4.36.2-py3-none-any.whl.metadata (126 kB)
Collecting kornia>=0.6 (from imaginairy)
  Using cached kornia-0.7.1-py2.py3-none-any.whl.metadata (12 kB)
Collecting uvicorn>=0.16.0 (from imaginairy)
  Downloading uvicorn-0.26.0-py3-none-any.whl.metadata (6.4 kB)
INFO: pip is looking at multiple versions of imaginairy to determine which version is compatible with other requirements. This could take a while.
Collecting imaginairy
  Using cached imaginAIry-14.0.3-py3-none-win_amd64.whl.metadata (36 kB)
Collecting fairscale>=0.4.4 (from imaginairy)
  Using cached fairscale-0.4.13-py3-none-any.whl
Collecting imaginairy
  Using cached imaginAIry-14.0.2-py3-none-win_amd64.whl.metadata (36 kB)
  Using cached imaginAIry-14.0.1-py3-none-win_amd64.whl.metadata (36 kB)
  Using cached imaginAIry-14.0.0-py3-none-win_amd64.whl.metadata (53 kB)
  Using cached imaginAIry-13.2.1-py3-none-win_amd64.whl.metadata (48 kB)
Collecting torch<2.0.0,>=1.13.1 (from imaginairy)
  Using cached torch-1.13.1-cp310-cp310-win_amd64.whl (162.6 MB)
Collecting pytorch-lightning<2.0.0,>=1.4.2 (from imaginairy)
  Using cached pytorch_lightning-1.9.5-py3-none-any.whl (829 kB)
Collecting torchmetrics>=0.6.0 (from imaginairy)
  Using cached torchmetrics-1.3.0.post0-py3-none-any.whl.metadata (20 kB)
Collecting imaginairy
  Using cached imaginAIry-13.2.0-py3-none-win_amd64.whl.metadata (48 kB)
Collecting xformers>=0.0.16 (from imaginairy)
  Using cached xformers-0.0.23.post1-cp310-cp310-win_amd64.whl.metadata (1.0 kB)
Collecting colorama (from click>=8.0.0->imaginairy)
  Using cached colorama-0.4.6-py2.py3-none-any.whl (25 kB)
Collecting importlib-metadata (from diffusers>=0.3.0->imaginairy)
  Using cached importlib_metadata-7.0.1-py3-none-any.whl.metadata (4.9 kB)
Requirement already satisfied: filelock in c:\users\anon\anaconda3\envs\imagine\lib\site-packages (from diffusers>=0.3.0->imaginairy) (3.9.0)
Collecting huggingface-hub>=0.20.2 (from diffusers>=0.3.0->imaginairy)
  Using cached huggingface_hub-0.20.2-py3-none-any.whl.metadata (12 kB)
Collecting regex!=2019.12.17 (from diffusers>=0.3.0->imaginairy)
  Using cached regex-2023.12.25-cp310-cp310-win_amd64.whl.metadata (41 kB)
Collecting filterpy (from facexlib>=0.2.1.1->imaginairy)
  Using cached filterpy-1.4.5-py3-none-any.whl
Collecting numba (from facexlib>=0.2.1.1->imaginairy)
  Using cached numba-0.58.1-cp310-cp310-win_amd64.whl.metadata (2.8 kB)
Collecting starlette<0.36.0,>=0.35.0 (from fastapi>=0.70.0->imaginairy)
  Downloading starlette-0.35.1-py3-none-any.whl.metadata (5.8 kB)
Collecting typing-extensions>=4.8.0 (from fastapi>=0.70.0->imaginairy)
  Downloading typing_extensions-4.9.0-py3-none-any.whl.metadata (3.0 kB)
Collecting wcwidth<0.3.0,>=0.2.12 (from ftfy>=6.0.1->imaginairy)
  Downloading wcwidth-0.2.13-py2.py3-none-any.whl.metadata (14 kB)
Collecting packaging (from kornia>=0.6->imaginairy)
  Using cached packaging-23.2-py3-none-any.whl.metadata (3.2 kB)
Collecting antlr4-python3-runtime==4.9.* (from omegaconf>=2.1.1->imaginairy)
  Using cached antlr4_python3_runtime-4.9.3-py3-none-any.whl
Collecting PyYAML>=5.1.0 (from omegaconf>=2.1.1->imaginairy)
  Downloading PyYAML-6.0.1-cp310-cp310-win_amd64.whl.metadata (2.1 kB)
Collecting sentencepiece (from open-clip-torch>=2.0.0->imaginairy)
  Using cached sentencepiece-0.1.99-cp310-cp310-win_amd64.whl (977 kB)
Collecting annotated-types>=0.4.0 (from pydantic>=2.3.0->imaginairy)
  Downloading annotated_types-0.6.0-py3-none-any.whl.metadata (12 kB)
Collecting pydantic-core==2.14.6 (from pydantic>=2.3.0->imaginairy)
  Downloading pydantic_core-2.14.6-cp310-none-win_amd64.whl.metadata (6.6 kB)
Requirement already satisfied: fsspec>2021.06.0 in c:\users\anon\anaconda3\envs\imagine\lib\site-packages (from fsspec[http]>2021.06.0->pytorch-lightning<2.0.0,>=1.4.2->imaginairy) (2023.4.0)
Collecting lightning-utilities>=0.6.0.post0 (from pytorch-lightning<2.0.0,>=1.4.2->imaginairy)
  Using cached lightning_utilities-0.10.0-py3-none-any.whl.metadata (4.8 kB)
Requirement already satisfied: charset-normalizer<3,>=2 in c:\users\anon\anaconda3\envs\imagine\lib\site-packages (from requests>=2.28.1->imaginairy) (2.1.1)
Requirement already satisfied: idna<4,>=2.5 in c:\users\anon\anaconda3\envs\imagine\lib\site-packages (from requests>=2.28.1->imaginairy) (3.4)
Requirement already satisfied: urllib3<1.27,>=1.21.1 in c:\users\anon\anaconda3\envs\imagine\lib\site-packages (from requests>=2.28.1->imaginairy) (1.26.13)
Requirement already satisfied: certifi>=2017.4.17 in c:\users\anon\anaconda3\envs\imagine\lib\site-packages (from requests>=2.28.1->imaginairy) (2022.12.7)
INFO: pip is looking at multiple versions of torchvision to determine which version is compatible with other requirements. This could take a while.
Collecting torchvision>=0.13.1 (from imaginairy)
  Using cached torchvision-0.16.2-cp310-cp310-win_amd64.whl.metadata (6.6 kB)
  Using cached torchvision-0.16.1-cp310-cp310-win_amd64.whl.metadata (6.6 kB)
  Using cached torchvision-0.16.0-cp310-cp310-win_amd64.whl.metadata (6.6 kB)
  Using cached torchvision-0.15.2-cp310-cp310-win_amd64.whl (1.2 MB)
  Using cached torchvision-0.15.1-cp310-cp310-win_amd64.whl (1.2 MB)
  Using cached torchvision-0.14.1-cp310-cp310-win_amd64.whl (1.1 MB)
Collecting tokenizers<0.19,>=0.14 (from transformers>=4.19.2->imaginairy)
  Using cached tokenizers-0.15.0-cp310-none-win_amd64.whl.metadata (6.8 kB)
Collecting h11>=0.8 (from uvicorn>=0.16.0->imaginairy)
  Downloading h11-0.14.0-py3-none-any.whl (58 kB)
     ---------------------------------------- 58.3/58.3 kB ? eta 0:00:00
INFO: pip is looking at multiple versions of xformers to determine which version is compatible with other requirements. This could take a while.
Collecting xformers>=0.0.16 (from imaginairy)
  Using cached xformers-0.0.23-cp310-cp310-win_amd64.whl.metadata (1.0 kB)
  Using cached xformers-0.0.22.post7-cp310-cp310-win_amd64.whl.metadata (1.0 kB)
  Using cached xformers-0.0.22.post4-cp310-cp310-win_amd64.whl.metadata (1.0 kB)
  Using cached xformers-0.0.22-cp310-cp310-win_amd64.whl.metadata (1.0 kB)
  Using cached xformers-0.0.21-cp310-cp310-win_amd64.whl.metadata (1.0 kB)
  Using cached xformers-0.0.20-cp310-cp310-win_amd64.whl.metadata (1.1 kB)
Collecting pyre-extensions==0.0.29 (from xformers>=0.0.16->imaginairy)
  Using cached pyre_extensions-0.0.29-py3-none-any.whl (12 kB)
Collecting xformers>=0.0.16 (from imaginairy)
  Using cached xformers-0.0.19-cp310-cp310-win_amd64.whl (96.7 MB)
INFO: pip is still looking at multiple versions of xformers to determine which version is compatible with other requirements. This could take a while.
  Using cached xformers-0.0.18-cp310-cp310-win_amd64.whl (112.3 MB)
Collecting pyre-extensions==0.0.23 (from xformers>=0.0.16->imaginairy)
  Using cached pyre_extensions-0.0.23-py3-none-any.whl (11 kB)
Collecting xformers>=0.0.16 (from imaginairy)
  Using cached xformers-0.0.17-cp310-cp310-win_amd64.whl (112.6 MB)
  Using cached xformers-0.0.16-cp310-cp310-win_amd64.whl (40.0 MB)
Collecting typing-inspect (from pyre-extensions==0.0.23->xformers>=0.0.16->imaginairy)
  Using cached typing_inspect-0.9.0-py3-none-any.whl.metadata (1.5 kB)
Collecting aiohttp!=4.0.0a0,!=4.0.0a1 (from fsspec[http]>2021.06.0->pytorch-lightning<2.0.0,>=1.4.2->imaginairy)
  Downloading aiohttp-3.9.1-cp310-cp310-win_amd64.whl.metadata (7.6 kB)
INFO: pip is looking at multiple versions of huggingface-hub to determine which version is compatible with other requirements. This could take a while.
Collecting fsspec[http]>2021.06.0 (from pytorch-lightning<2.0.0,>=1.4.2->imaginairy)
  Downloading fsspec-2023.12.2-py3-none-any.whl.metadata (6.8 kB)
Requirement already satisfied: setuptools in c:\users\anon\anaconda3\envs\imagine\lib\site-packages (from lightning-utilities>=0.6.0.post0->pytorch-lightning<2.0.0,>=1.4.2->imaginairy) (68.2.2)
Collecting anyio<5,>=3.4.0 (from starlette<0.36.0,>=0.35.0->fastapi>=0.70.0->imaginairy)
  Downloading anyio-4.2.0-py3-none-any.whl.metadata (4.6 kB)
Collecting matplotlib (from filterpy->facexlib>=0.2.1.1->imaginairy)
  Downloading matplotlib-3.8.2-cp310-cp310-win_amd64.whl.metadata (5.9 kB)
Collecting zipp>=0.5 (from importlib-metadata->diffusers>=0.3.0->imaginairy)
  Using cached zipp-3.17.0-py3-none-any.whl.metadata (3.7 kB)
Collecting llvmlite<0.42,>=0.41.0dev0 (from numba->facexlib>=0.2.1.1->imaginairy)
  Using cached llvmlite-0.41.1-cp310-cp310-win_amd64.whl.metadata (4.9 kB)
Collecting attrs>=17.3.0 (from aiohttp!=4.0.0a0,!=4.0.0a1->fsspec[http]>2021.06.0->pytorch-lightning<2.0.0,>=1.4.2->imaginairy)
  Downloading attrs-23.2.0-py3-none-any.whl.metadata (9.5 kB)
Collecting multidict<7.0,>=4.5 (from aiohttp!=4.0.0a0,!=4.0.0a1->fsspec[http]>2021.06.0->pytorch-lightning<2.0.0,>=1.4.2->imaginairy)
  Downloading multidict-6.0.4-cp310-cp310-win_amd64.whl (28 kB)
Collecting yarl<2.0,>=1.0 (from aiohttp!=4.0.0a0,!=4.0.0a1->fsspec[http]>2021.06.0->pytorch-lightning<2.0.0,>=1.4.2->imaginairy)
  Downloading yarl-1.9.4-cp310-cp310-win_amd64.whl.metadata (32 kB)
Collecting frozenlist>=1.1.1 (from aiohttp!=4.0.0a0,!=4.0.0a1->fsspec[http]>2021.06.0->pytorch-lightning<2.0.0,>=1.4.2->imaginairy)
  Downloading frozenlist-1.4.1-cp310-cp310-win_amd64.whl.metadata (12 kB)
Collecting aiosignal>=1.1.2 (from aiohttp!=4.0.0a0,!=4.0.0a1->fsspec[http]>2021.06.0->pytorch-lightning<2.0.0,>=1.4.2->imaginairy)
  Downloading aiosignal-1.3.1-py3-none-any.whl (7.6 kB)
Collecting async-timeout<5.0,>=4.0 (from aiohttp!=4.0.0a0,!=4.0.0a1->fsspec[http]>2021.06.0->pytorch-lightning<2.0.0,>=1.4.2->imaginairy)
  Downloading async_timeout-4.0.3-py3-none-any.whl.metadata (4.2 kB)
Collecting sniffio>=1.1 (from anyio<5,>=3.4.0->starlette<0.36.0,>=0.35.0->fastapi>=0.70.0->imaginairy)
  Downloading sniffio-1.3.0-py3-none-any.whl (10 kB)
Collecting exceptiongroup>=1.0.2 (from anyio<5,>=3.4.0->starlette<0.36.0,>=0.35.0->fastapi>=0.70.0->imaginairy)
  Downloading exceptiongroup-1.2.0-py3-none-any.whl.metadata (6.6 kB)
Collecting contourpy>=1.0.1 (from matplotlib->filterpy->facexlib>=0.2.1.1->imaginairy)
  Downloading contourpy-1.2.0-cp310-cp310-win_amd64.whl.metadata (5.8 kB)
Collecting cycler>=0.10 (from matplotlib->filterpy->facexlib>=0.2.1.1->imaginairy)
  Downloading cycler-0.12.1-py3-none-any.whl.metadata (3.8 kB)
Collecting fonttools>=4.22.0 (from matplotlib->filterpy->facexlib>=0.2.1.1->imaginairy)
  Downloading fonttools-4.47.2-cp310-cp310-win_amd64.whl.metadata (160 kB)
     ---------------------------------------- 160.8/160.8 kB 4.7 MB/s eta 0:00:00
Collecting kiwisolver>=1.3.1 (from matplotlib->filterpy->facexlib>=0.2.1.1->imaginairy)
  Downloading kiwisolver-1.4.5-cp310-cp310-win_amd64.whl.metadata (6.5 kB)
Collecting pyparsing>=2.3.1 (from matplotlib->filterpy->facexlib>=0.2.1.1->imaginairy)
  Downloading pyparsing-3.1.1-py3-none-any.whl.metadata (5.1 kB)
Collecting python-dateutil>=2.7 (from matplotlib->filterpy->facexlib>=0.2.1.1->imaginairy)
  Downloading python_dateutil-2.8.2-py2.py3-none-any.whl (247 kB)
     ---------------------------------------- 247.7/247.7 kB 15.8 MB/s eta 0:00:00
Collecting mypy-extensions>=0.3.0 (from typing-inspect->pyre-extensions==0.0.23->xformers>=0.0.16->imaginairy)
  Using cached mypy_extensions-1.0.0-py3-none-any.whl (4.7 kB)
Collecting six>=1.5 (from python-dateutil>=2.7->matplotlib->filterpy->facexlib>=0.2.1.1->imaginairy)
  Downloading six-1.16.0-py2.py3-none-any.whl (11 kB)
Using cached imaginAIry-13.2.0-py3-none-win_amd64.whl (7.9 MB)
Downloading click-8.1.7-py3-none-any.whl (97 kB)
   ---------------------------------------- 97.9/97.9 kB ? eta 0:00:00
Using cached click_help_colors-0.9.4-py3-none-any.whl (6.4 kB)
Using cached diffusers-0.25.1-py3-none-any.whl (1.8 MB)
Using cached einops-0.7.0-py3-none-any.whl (44 kB)
Downloading fastapi-0.109.0-py3-none-any.whl (92 kB)
   ---------------------------------------- 92.0/92.0 kB 5.1 MB/s eta 0:00:00
Using cached ftfy-6.1.3-py3-none-any.whl (53 kB)
Downloading imageio-2.33.1-py3-none-any.whl (313 kB)
   ---------------------------------------- 313.3/313.3 kB 20.2 MB/s eta 0:00:00
Using cached kornia-0.7.1-py2.py3-none-any.whl (756 kB)
Using cached open_clip_torch-2.24.0-py3-none-any.whl (1.5 MB)
Downloading opencv_python-4.9.0.80-cp37-abi3-win_amd64.whl (38.6 MB)
   ---------------------------------------- 38.6/38.6 MB 65.2 MB/s eta 0:00:00
Downloading protobuf-4.25.2-cp310-abi3-win_amd64.whl (413 kB)
   ---------------------------------------- 413.4/413.4 kB 25.2 MB/s eta 0:00:00
Downloading psutil-5.9.7-cp37-abi3-win_amd64.whl (252 kB)
   ---------------------------------------- 252.2/252.2 kB 15.1 MB/s eta 0:00:00
Downloading pydantic-2.5.3-py3-none-any.whl (381 kB)
   ---------------------------------------- 381.9/381.9 kB ? eta 0:00:00
Downloading pydantic_core-2.14.6-cp310-none-win_amd64.whl (1.9 MB)
   ---------------------------------------- 1.9/1.9 MB 58.8 MB/s eta 0:00:00
Using cached safetensors-0.4.1-cp310-none-win_amd64.whl (277 kB)
Using cached timm-0.9.12-py3-none-any.whl (2.2 MB)
Using cached torchmetrics-1.3.0.post0-py3-none-any.whl (840 kB)
Using cached tqdm-4.66.1-py3-none-any.whl (78 kB)
Using cached transformers-4.36.2-py3-none-any.whl (8.2 MB)
Downloading uvicorn-0.26.0-py3-none-any.whl (60 kB)
   ---------------------------------------- 60.5/60.5 kB ? eta 0:00:00
Downloading annotated_types-0.6.0-py3-none-any.whl (12 kB)
Downloading fsspec-2023.12.2-py3-none-any.whl (168 kB)
   ---------------------------------------- 169.0/169.0 kB ? eta 0:00:00
Using cached huggingface_hub-0.20.2-py3-none-any.whl (330 kB)
Using cached lightning_utilities-0.10.0-py3-none-any.whl (24 kB)
Using cached packaging-23.2-py3-none-any.whl (53 kB)
Downloading PyYAML-6.0.1-cp310-cp310-win_amd64.whl (145 kB)
   ---------------------------------------- 145.3/145.3 kB 9.0 MB/s eta 0:00:00
Using cached regex-2023.12.25-cp310-cp310-win_amd64.whl (269 kB)
Downloading starlette-0.35.1-py3-none-any.whl (71 kB)
   ---------------------------------------- 71.1/71.1 kB 4.1 MB/s eta 0:00:00
Using cached tokenizers-0.15.0-cp310-none-win_amd64.whl (2.2 MB)
Downloading typing_extensions-4.9.0-py3-none-any.whl (32 kB)
Downloading wcwidth-0.2.13-py2.py3-none-any.whl (34 kB)
Using cached importlib_metadata-7.0.1-py3-none-any.whl (23 kB)
Using cached numba-0.58.1-cp310-cp310-win_amd64.whl (2.6 MB)
Downloading aiohttp-3.9.1-cp310-cp310-win_amd64.whl (364 kB)
   ---------------------------------------- 364.6/364.6 kB ? eta 0:00:00
Downloading anyio-4.2.0-py3-none-any.whl (85 kB)
   ---------------------------------------- 85.5/85.5 kB 4.7 MB/s eta 0:00:00
Using cached llvmlite-0.41.1-cp310-cp310-win_amd64.whl (28.1 MB)
Using cached zipp-3.17.0-py3-none-any.whl (7.4 kB)
Downloading matplotlib-3.8.2-cp310-cp310-win_amd64.whl (7.6 MB)
   ---------------------------------------- 7.6/7.6 MB 81.2 MB/s eta 0:00:00
Using cached typing_inspect-0.9.0-py3-none-any.whl (8.8 kB)
Downloading async_timeout-4.0.3-py3-none-any.whl (5.7 kB)
Downloading attrs-23.2.0-py3-none-any.whl (60 kB)
   ---------------------------------------- 60.8/60.8 kB 3.2 MB/s eta 0:00:00
Downloading contourpy-1.2.0-cp310-cp310-win_amd64.whl (186 kB)
   ---------------------------------------- 186.7/186.7 kB 11.0 MB/s eta 0:00:00
Downloading cycler-0.12.1-py3-none-any.whl (8.3 kB)
Downloading exceptiongroup-1.2.0-py3-none-any.whl (16 kB)
Downloading fonttools-4.47.2-cp310-cp310-win_amd64.whl (2.2 MB)
   ---------------------------------------- 2.2/2.2 MB 69.9 MB/s eta 0:00:00
Downloading frozenlist-1.4.1-cp310-cp310-win_amd64.whl (50 kB)
   ---------------------------------------- 50.4/50.4 kB ? eta 0:00:00
Downloading kiwisolver-1.4.5-cp310-cp310-win_amd64.whl (56 kB)
   ---------------------------------------- 56.1/56.1 kB 2.9 MB/s eta 0:00:00
Downloading pyparsing-3.1.1-py3-none-any.whl (103 kB)
   ---------------------------------------- 103.1/103.1 kB 6.2 MB/s eta 0:00:00
Downloading yarl-1.9.4-cp310-cp310-win_amd64.whl (76 kB)
   ---------------------------------------- 76.4/76.4 kB 4.1 MB/s eta 0:00:00
Installing collected packages: wcwidth, sentencepiece, antlr4-python3-runtime, zipp, typing-extensions, sniffio, six, scipy, safetensors, regex, PyYAML, pyparsing, psutil, protobuf, packaging, opencv-python, mypy-extensions, multidict, llvmlite, kiwisolver, imageio, h11, ftfy, fsspec, frozenlist, fonttools, exceptiongroup, einops, cycler, contourpy, colorama, attrs, async-timeout, annotated-types, yarl, typing-inspect, tqdm, torch, python-dateutil, pydantic-core, omegaconf, numba, lightning-utilities, importlib-metadata, click, anyio, aiosignal, uvicorn, torchvision, torchmetrics, torchdiffeq, starlette, pyre-extensions, pydantic, matplotlib, kornia, huggingface-hub, fairscale, click-shell, click-help-colors, aiohttp, xformers, tokenizers, timm, filterpy, fastapi, diffusers, transformers, pytorch-lightning, open-clip-torch, facexlib, imaginairy
  Attempting uninstall: typing-extensions
    Found existing installation: typing_extensions 4.4.0
    Uninstalling typing_extensions-4.4.0:
      Successfully uninstalled typing_extensions-4.4.0
  Attempting uninstall: fsspec
    Found existing installation: fsspec 2023.4.0
    Uninstalling fsspec-2023.4.0:
      Successfully uninstalled fsspec-2023.4.0
  Attempting uninstall: torch
    Found existing installation: torch 2.1.2+cu118
    Uninstalling torch-2.1.2+cu118:
      Successfully uninstalled torch-2.1.2+cu118
  Attempting uninstall: torchvision
    Found existing installation: torchvision 0.16.2+cu118
    Uninstalling torchvision-0.16.2+cu118:
      Successfully uninstalled torchvision-0.16.2+cu118
ERROR: pip's dependency resolver does not currently take into account all the packages that are installed. This behaviour is the source of the following dependency conflicts.
torchaudio 2.1.2+cu118 requires torch==2.1.2+cu118, but you have torch 1.13.1 which is incompatible.
Successfully installed PyYAML-6.0.1 aiohttp-3.9.1 aiosignal-1.3.1 annotated-types-0.6.0 antlr4-python3-runtime-4.9.3 anyio-4.2.0 async-timeout-4.0.3 attrs-23.2.0 click-8.1.7 click-help-colors-0.9.4 click-shell-2.1 colorama-0.4.6 contourpy-1.2.0 cycler-0.12.1 diffusers-0.25.1 einops-0.7.0 exceptiongroup-1.2.0 facexlib-0.3.0 fairscale-0.4.13 fastapi-0.109.0 filterpy-1.4.5 fonttools-4.47.2 frozenlist-1.4.1 fsspec-2023.12.2 ftfy-6.1.3 h11-0.14.0 huggingface-hub-0.20.2 imageio-2.33.1 imaginairy-13.2.0 importlib-metadata-7.0.1 kiwisolver-1.4.5 kornia-0.7.1 lightning-utilities-0.10.0 llvmlite-0.41.1 matplotlib-3.8.2 multidict-6.0.4 mypy-extensions-1.0.0 numba-0.58.1 omegaconf-2.3.0 open-clip-torch-2.24.0 opencv-python-4.9.0.80 packaging-23.2 protobuf-4.25.2 psutil-5.9.7 pydantic-2.5.3 pydantic-core-2.14.6 pyparsing-3.1.1 pyre-extensions-0.0.23 python-dateutil-2.8.2 pytorch-lightning-1.9.5 regex-2023.12.25 safetensors-0.4.1 scipy-1.10.1 sentencepiece-0.1.99 six-1.16.0 sniffio-1.3.0 starlette-0.35.1 timm-0.9.12 tokenizers-0.15.0 torch-1.13.1 torchdiffeq-0.2.3 torchmetrics-1.3.0.post0 torchvision-0.14.1 tqdm-4.66.1 transformers-4.36.2 typing-extensions-4.9.0 typing-inspect-0.9.0 uvicorn-0.26.0 wcwidth-0.2.13 xformers-0.0.16 yarl-1.9.4 zipp-3.17.0

**Comment by brycedrennan** [comment]
```
INFO: pip is looking at multiple versions of imaginairy to determine which version is compatible with other requirements. This could take a while.
Collecting imaginairy
...
Using cached imaginAIry-13.2.1-py3-none-win_amd64.whl.metadata (48 kB)
```
hmmm yeah that's weird. why is it doing that.

**Comment by brycedrennan** [comment]
Looks like I have some changes to the requirements I haven't released yet. Maybe they'll help with this. I'll make a release right now.

**Comment by brycedrennan** [comment]
Okay I just released 4.1.0 - no idea whether it will help but it does have fewer dependencies, so it's possible. Let me know if it still doesnt work and I'm willing to look into this more.

**Comment by brycedrennan** [comment]
Running `pip install imaginairy==14.0.4 --verbose` might provide more information about whats going on.  You could also make sure pip is upgraded with `pip install pip --upgrade`
